bash3/decode.sh

```diff
@@ -0,0 +1,14 @@
+# Script to decode Base64 text
+
+#!/bin/bash
+
+encoded_string="$1"
+
+if [ -z "$encoded_string" ]; then
+  echo "Usage: $0 <encoded_string>"
+  exit 1
+fi
+
+decoded_string=$(echo "$encoded_string" | base64 -d)
```

Comment:
```suggestion
base64 -d <<< "$encoded_string"
echo  # tests expect a newline at the end
```

(and remove `echo "$decoded_string"` below)